src/api/Gemfile

```diff
@@ -64,7 +64,7 @@ gem 'peek-dalli'
 gem 'peek-mysql2'
 # for kerberos authentication
 gem "gssapi", require: false
-
+gem 'mailcatcher'
```

Comment:
@evanrolfe, this should be in `group :development` because it's not needed for production.